app/controllers/api/account.php

```diff
@@ -519,7 +522,7 @@
             'countryCode' => ($record) ? \strtolower($record['country']['iso_code']) : '--',
```

Comment:
I still think we should change the default create session endpoint to specifically create a session with email. Currently it seems like the API has a bias and suggest email is *the* way to go.

Reply:
We can do that 👍 I initially did this for less breaking change and less work for me 😉